### ISSUE TITLE
Title option should handle a function

### ISSUE THREAD

**Issue by Hiswe** [Hiswe/gulp-svg-symbols#26]
For finer accessibility, we should be able to provide a custom string for each symbol

**Comment by Hiswe** [comment]
In order to keep gulp-svg-symbols as dumb as possible, this king of thing should be handle on an optimisation task before calling svg-symbols.

It event should be nicer to not have this option anymore
